core/controllers/admin_test.py

```diff
@@ -650,7 +646,7 @@ def test_update_flag_rules_with_unknown_feature_name_returns_400(self):
             response = self.post_json(
                 '/adminhandler', {
                     'action': 'update_feature_flag_rules',
-                    'feature_name': 'test_feature_1',
+                    'feature_name': 'TEST_FEATURE_1',
```

Comment:
No, here it should be kept in lowercase.

Reply:
Made the require change in the new commit @vojtechjelinek